platform/ui/src/components/userPreferencesModal/AboutModal.js

```diff
@@ -1,14 +1,16 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Modal from 'react-bootstrap-modal';
+import detect from 'browser-detect';
 import './AboutModal.styl';
+import { withTranslation } from '../../utils/LanguageProvider';
 
 import 'react-bootstrap-modal/lib/css/rbm-patch.css';
 
 // TODO: Is this the only component importing these?
 import './../../design/styles/common/modal.styl';
 
-export class AboutModal extends Component {
+class AboutModal extends Component {
```

Comment:
Not worth doing for this PR, but simple/dumb components can probably just be Function components